build.sbt

```diff
@@ -8,7 +8,9 @@ version := "0.0.2"
 
 testOptions in ThisBuild += Tests.Argument(TestFrameworks.JUnit, "-v")
 
-resolvers in ThisBuild += "NCSA Sonatype Releases" at "https://opensource.ncsa.illinois.edu/nexus/content/repositories/releases"
+resolvers in ThisBuild ++= Seq(
+  "NCSA Sonatype Releases" at "https://opensource.ncsa.illinois.edu/nexus/content/repositories/releases"
+)
 
 libraryDependencies in ThisBuild := Seq(
   "junit" % "junit" % "4.11" % "test",
```

Comment:
Can you compare this file with the build.sbt at [Standard DFDL Schema Project Layout](https://opensource.ncsa.illinois.edu/confluence/display/DFDL/Standard+DFDL+Schema+Project+Layout)? I think there's a few things that should be here (e.g. name should start with "dfdl-", crossPaths should be set to false, should include the snapshot resolver, etc.).